dockers/base-cuda/Dockerfile

```diff
@@ -83,21 +83,21 @@ RUN \
     python ./requirements/adjust-versions.py requirements/extra.txt ${PYTORCH_VERSION} && \
     python ./requirements/adjust-versions.py requirements/examples.txt ${PYTORCH_VERSION} && \
     python -c "print(' '.join([ln for ln in open('requirements/extra.txt').readlines() if 'horovod' in ln]))" > ./requirements/horovod.txt && \
-    python assistant.py requirements_prune_pkgs requirements/examples.txt "horovod" && \
+    python assistant.py requirements_prune_pkgs requirements/extra.txt "horovod" && \
```

Comment:
FYI, this was the bug...
@awaelchli @akihironitta

Reply:
That's why reinstalling it fixed the issue in my PR! Thanks for looking into this!!! :)

Reply:
yeah, but why fix the consecutive issue when we can address the source/original one :)